ionic2/app/components/poke-details/poke-type.component.ts

```diff
@@ -1,12 +1,54 @@
-import { Component} from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 
 @Component({
   template: require('./poke-type.component.html'),
-  selector: 'poke-type'
+  selector: 'poke-type',
+  styles: [`
```

Comment:
I know it's now a lot but I think it's more consistent to move the CSS to a different file rather than inlining it.